jest.config.js

```diff
@@ -46,5 +46,9 @@ module.exports = {
     '.svg': '<rootDir>/__mocks__/svgTransformer.js',
   },
 
+  // Ignore transforming node_modules except for:
+  // 1. `react-children-utilities`
+  transformIgnorePatterns: ['/node_modules/(?!(react-children-utilities)/)'],
```

Comment:
Without it I get the following error when running the tests:

```bash
Jest encountered an unexpected token
```
<details>
  <summary>Error Details</summary>
  <p>
    
Jest encountered an unexpected token

    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.

    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".

    Here's what you can do:
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/en/configuration.html

    Details:

    /Users/matt/Documents/projects/mongodb/leafygreen-ui/node_modules/react-children-utilities/react-children-utilities.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){import { Children } from 'react';
                                                                                                    ^

    SyntaxError: Unexpected token {

      1 | import React, { ReactNode } from 'react';
    > 2 | import { onlyText } from 'react-children-utilities';
        | ^
      3 | import flatMap from 'lodash/flatMap';
      4 | 
      5 | /**

      at ScriptTransformer._transformAndBuildScript (node_modules/jest-runtime/build/script_transformer.js:403:17)
      at Object.<anonymous> (packages/pipeline/src/utils/getPipelineCounterTooltip.ts:2:1)
  </p>
</details>

I think the third-party package is not setup correctly with the `module` syntax. Both their `module` and `main` definitions in `package.json` point to the same `react-children-utilities.js` file, which for whatever reason is not playing nicely with the build.

Adding this Jest configuration resolves the issue.

Reply:
Damn, that's an amazing error message.